testcases/peak/ckan_update_metadata_1u_1s.js

```diff
@@ -0,0 +1,44 @@
+import http from 'k6/http';
+import {HTTP_OPTIONS} from "../../config/httpConfig.js";
+import { CONFIG } from '../../config/config.js';
+import {getUrl, findOrCreateResource, deleteAllRessources} from "../../util/resources.js";
+import {createPeakScenarios} from "./scenarios.js";
+
+export let options = CONFIG.options;
+
+const URL = getUrl();
+const CKAN_API_URL = URL + CONFIG.ckanAPIPath;
+const DATASET_NAME = CONFIG.datasetName;
+const RESOURCE_NAME = CONFIG.resourceName;
+let scenarioDuration = 300;
+let scenarioPause = 30;
+let scenarioNamePrefix = `${__ENV.SYSTEM}_update_metadata_1u_1s`;
+
+let vus = [1, 2, 4, 6, 10, 42];
```

Comment:
Warum weichen hier die vus von den vus (1, 2, 4, **_8, 16, 32, 48_**) in den anderen Tests ab?